src/test/mochitest/examples/sourcemapped/fixtures/classes/input.js

```diff
@@ -7,10 +7,15 @@ export default function root() {
     }
   }
 
+
+
   class Another {
+    bound = () => {
```

Comment:
I think the build script needs to be run again, since this isn't reflected in the output files.